rita/src/rita_client/dashboard/network_endpoints.rs

```diff
@@ -3,6 +3,8 @@ use actix_web::*;
 
 use futures::Future;
 
+use failure::Error;
```

Comment:
Is this really necessary? This is the only change in the file.

Reply:
yes, its a result of a related change which borked type inference